Gemfile

```diff
@@ -1,9 +1,14 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-gem 'inspec', '~> 3'
-gem 'rake'
-gem 'rubocop'
+gem 'github_changelog_generator', '~> 1.12.0', :source => 'https://rubygems.org/'
+gem 'rake', :source => 'https://rubygems.org/'
+gem 'rubocop', :source => 'https://rubygems.org/'
+gem 'unf_ext', :source => 'https://rubygems.org/'
 
-group :tools do
-  gem 'github_changelog_generator', '~> 1.12.0'
+source 'https://packagecloud.io/cinc-project/stable' do
+  gem "chef-config"
```

Comment:
We still need to fix the robucop issues as reported by travis https://travis-ci.org/github/dev-sec/cis-dil-benchmark/builds/717146821?utm_source=github_status&utm_medium=notification

```
Offenses:
488
489Gemfile:3:48: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
490gem 'github_changelog_generator', '~> 1.12.0', :source => 'https://rubygems.org/'
491                                               ^^^^^^^^^^
492Gemfile:4:13: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
493gem 'rake', :source => 'https://rubygems.org/'
494            ^^^^^^^^^^
495Gemfile:5:16: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
496gem 'rubocop', :source => 'https://rubygems.org/'
497               ^^^^^^^^^^
498Gemfile:6:16: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
499gem 'unf_ext', :source => 'https://rubygems.org/'
500               ^^^^^^^^^^
501Gemfile:9:7: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
502  gem "chef-config"
503      ^^^^^^^^^^^^^
504Gemfile:10:7: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
505  gem "chef-utils"
506      ^^^^^^^^^^^^
507Gemfile:11:7: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
508  gem "cinc-auditor-bin"
509      ^^^^^^^^^^^^^^^^^^
510Gemfile:12:7: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
511  gem "inspec"
512      ^^^^^^^^
513Gemfile:13:7: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
514  gem "inspec-core"
515      ^^^^^^^^^^^^^
```